client/App/details/details.js

```diff
@@ -2,9 +2,61 @@ angular.module('moviedash.details', [])
 
 .controller('DetailsCtrl', ['$scope', 'selected', function ($scope, selected) {
   // Code
-
   $scope.movie = selected.getStorage('movie');
 
+  $scope.reminderMsg = 'LEAVE REMINDER';
+  var uniqueMovieId = $scope.movie.id + $scope.movie.showTime + $scope.movie.movieName;
+  var alreadyClicked = false;
+  var timeoutID;
+
+  // This block of code only executes when user chooses to use the movie reminder button
+  if (selected.getStorage(uniqueMovieId)) {
+    var reminderInfo = selected.getStorage(uniqueMovieId);
+ 
+    $scope.btn = reminderInfo[0];
+    alreadyClicked = reminderInfo[2];
+    timeoutID = reminderInfo[1];
+
+    if (alreadyClicked) {
+      $scope.reminderMsg = 'CLEAR REMINDER';
+    }
+  }
+
+  $scope.reminder = function(movie) {
+    // HTML 5 Notifications setup
+    var Notification = window.Notification || window.mozNotification || window.webkitNotification;
+    
+    Notification.requestPermission(function (permission) {
```

Comment:
Curious to see what happens when permission is denied. I'll test it after we merge.